### ISSUE TITLE
Adding a "Handling General Errors" section to the documentation.

### ISSUE THREAD

**Issue by yellowwoods12** [opencollective/opencollective#2122]
## User story 
 _While I was following the Readme Integration article provided in the documentation I got an error in the process. This gave me an idea of adding a "Handling General Errors" section to the documentation that involve steps of installation of any specific file or running any particular process._

## Best solution for this problem
_Adding a "Handling General Errors" section to the documentation of Open Collective.This will help the newbies to get a better understanding of the issues they might face and will also encourage participation in the community._

## Documentation 
_The goal of the documentation should be to make it easier for others to fulfill the motive for which that particular documentation article is focused on and this section will encourage this._

**Comment by carlcollins35** [comment]
Great Idea @yellowwoods12 , I can get started on that.

**Comment by Betree** [comment]
Sounds good to me!

**Comment by yellowwoods12** [comment]
@Betree Yes like we can test the installation over different OS distributions and create this section accordingly.
